talestation_modules/code/modules/antagonists/bloodsuckers/bloodsucker/bloodsucker_life.dm

```diff
@@ -0,0 +1,429 @@
+		switch(bloodsucker_blood_volume)
+			if(BLOOD_VOLUME_OKAY to INFINITY) // 336 and up, we are perfectly fine.
+				owner.current.blood_volume = initial(bloodsucker_blood_volume)
+			if(BLOOD_VOLUME_BAD to BLOOD_VOLUME_OKAY) // 224 to 336
+				owner.current.blood_volume = BLOOD_VOLUME_SAFE
+			else // 224 and below
+				owner.current.blood_volume = BLOOD_VOLUME_OKAY
+		return
+
+	owner.current.blood_volume = bloodsucker_blood_volume
+
+/// Gibs the Bloodsucker, roundremoving them.
+/datum/antagonist/bloodsucker/proc/FinalDeath()
+	// If we have no body, end here.
+	if(!owner.current || dust_timer)
+		return
+
+	FreeAllVassals()
+	DisableAllPowers(forced = TRUE)
+	if(!iscarbon(owner.current))
+		owner.current.gib(TRUE, FALSE, FALSE)
+		return
+	// Drop anything in us and play a tune
+	var/mob/living/carbon/user = owner.current
+	owner.current.drop_all_held_items()
+	owner.current.unequip_everything()
+	user.remove_all_embedded_objects()
+	playsound(owner.current, 'sound/effects/tendril_destroyed.ogg', 40, TRUE)
+
+	if(my_clan)
+		SEND_SIGNAL(my_clan, BLOODSUCKER_FINAL_DEATH, owner.current)
+
+// Bloodsuckers moodlets //
+/datum/mood_event/drankblood
+	description = "<span class='nicegreen'>I have fed greedly from that which nourishes me.</span>\n"
+	mood_change = 10
+	timeout = 8 MINUTES
+
+/datum/mood_event/drankblood_bad
+	description = "<span class='boldwarning'>I drank the blood of a lesser creature. Disgusting.</span>\n"
+	mood_change = -4
+	timeout = 3 MINUTES
+
+/datum/mood_event/drankblood_dead
+	description = "<span class='boldwarning'>I drank dead blood. I am better than this.</span>\n"
+	mood_change = -7
+	timeout = 8 MINUTES
+
+/datum/mood_event/drankblood_synth
+	description = "<span class='boldwarning'>I drank synthetic blood. What is wrong with me?</span>\n"
+	mood_change = -7
+	timeout = 8 MINUTES
+
+/datum/mood_event/drankkilled
+	description = "<span class='boldwarning'>I fed off of a dead person. I feel... less human.</span>\n"
+	mood_change = -15
+	timeout = 10 MINUTES
+
+/datum/mood_event/madevamp
+	description = "<span class='boldwarning'>A mortal has reached an apotheosis- undeath- by my own hand.</span>\n"
+	mood_change = 15
+	timeout = 20 MINUTES
+
+/datum/mood_event/coffinsleep
+	description = "<span class='nicegreen'>I slept in a coffin during the day. I feel whole again.</span>\n"
+	mood_change = 10
+	timeout = 6 MINUTES
+
+/datum/mood_event/daylight_1
+	description = "<span class='boldwarning'>I slept poorly in a makeshift coffin during the day.</span>\n"
+	mood_change = -3
+	timeout = 6 MINUTES
+
+/datum/mood_event/daylight_2
+	description = "<span class='boldwarning'>I have been scorched by the unforgiving rays of the sun.</span>\n"
+	mood_change = -6
+	timeout = 6 MINUTES
+
+///Candelabrum's mood event to non Bloodsucker/Vassals
+/datum/mood_event/vampcandle
+	description = "<span class='boldwarning'>Something is making your mind feel... loose.</span>\n"
+	mood_change = -15
+	timeout = 5 MINUTES
+
```

Comment:
A lot of the procs in this file are using PascalCase

Reply:
changed the unimportant ones, kept the important ones,

Reply:
I also, in some ocassions, kept PascalCase as a way to keep track of things I've touched, and parts that is still from Swain's era, though I wasn't consistent with that and lost track of it.